settings/payfort.py

```diff
@@ -1,3 +1,28 @@
 from ecommerce.settings.test import *
 
 INSTALLED_APPS += ['ecommerce_payfort']
+
+PAYMENT_PROCESSOR_CONFIG = {
+    'edx': {
+        'payfort': {
+            # 'access_token': '1234',
```

Comment:
it's all empty, no problem?

Reply:
these are for the tests, I'm working on it 👍🏼